src/index.js

```diff
@@ -0,0 +1,40 @@
+import './style.css';
+
+import { addTask, crud } from './modules/crud.js';
```

Comment:
You did a great job in this code, and it would be more order if delete the empty line between those `imports`.